.github/workflows/stale-issues.yml

```
@@ -0,0 +1,42 @@
+name: Check stale issues
+
+on:
+  workflow_dispatch: # FIXME: remove
+  schedule:
+    - cron: '0 12 * * *'
+
+env:
+  PERIOD: 14 # in days
+  WAIT_FOR_INFO: "waiting for more input"
+  DRY_RUN: true
+
+jobs:
+  check-issue-status:
+    name: Retrieve all open issues that are inactive
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Get inactive open issues
+        id: stale_issues
+        uses: ./.github/actions/stale_issues
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          period: ${{ env.PERIOD }}
+      - name: Comment on found issues
+        if: ${{ success() }}
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          for issue_url in ${{ steps.stale_issues.outputs.result }}; do
+            # Filter stale issues that were triaged and are not waiting for more information
+            stale_triaged=$(gh issue view $issue_url \
```

I'd say that it's not stale if we decide to exclude it, e.g. due to being/not being stale.
```suggestion
            is_stale=$(gh issue view $issue_url \
```

---

By the way, by "triaged" do you mean simply not having the wait label? Because I don't see any other filtering here.

I think that we should consider an issue triaged when it has impact and effort label.

Also, for the purpose of backlog closing (https://github.com/ethereum/solidity/issues/13615#issuecomment-1325635999), we'll want to include both triaged issues and ones waiting for input.

---

Yeah. Having any label except the wait label (i.e., the issue has been triaged if we labeled it and it is not depending on the op for more input).

I decided to change the other action (see here: https://github.com/ethereum/solidity/pull/13745/commits/7ff369a752b07c97b1c495f506f94d22632cf6e2) to also work on issues, so this one here could be repurpose or discarded.